Library/OcLogAggregatorLibNull/OcLogAggregatorLibNull.inf

```diff
@@ -0,0 +1,25 @@
+## @file
```

Comment:
What is the point of this change?

Reply:
I don't need this to fix the screen clear now, and an going to do a revised version of this PR without it. But I still believe this is wrong and should be fixed to save someone else having to work it out in future.

The Null and Serial versions of DebugLib also include null functions for OcLogAggregatorLib. But OcLogAggregator lib is a separate library from DebugLib.

Because of further library dependencies, this error (as I believe it to be) was making it impossible to link OcAfterBootCompatLib anywhere except where it was already linked. After these changes - which look correct to me anyway - I could link it to another another library, OcBootManagementLib, as I needed to in this current (to be replaced) version of the PR, just fine.